src/openloco/ui/WindowManager.cpp

```diff
@@ -1824,6 +1824,93 @@ namespace openloco::ui::WindowManager
         }
     }
 
+    /**
+     * 0x004A0A18
+     *
+     * @param flags @<al>
+     */
+    void viewportSetVisibility(uint8_t flags)
+    {
+        auto window = WindowManager::getMainWindow();
+
+        if (window == nullptr)
+            return;
+
+        auto viewport = window->viewports[0];
+        bool flagsChanged = false;
+        if (flags < 1 || flags > 3)
```

Comment:
Make this a switch statement where `flags < 1 || flags > 3` is default (aka 0 and >3). Then you have `2` and `3`